2023-12-03 00:06:07,692 - rendered_api.py -> requests - line:70 - ERROR: 动态渲染请求出现错误, 错误内容是ImplementationClass.create_browser_context_page() got an unexpected keyword argument 'is_block_image'
2023-12-03 00:15:34,571 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:15:34,574 - rendered_api.py -> requests - line:79 - ERROR: 动态渲染请求出现错误, 错误内容是Object of type coroutine is not JSON serializable
2023-12-03 00:16:06,372 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:16:06,373 - rendered_api.py -> requests - line:79 - ERROR: 动态渲染请求出现错误, 错误内容是Object of type coroutine is not JSON serializable
2023-12-03 00:17:00,746 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:18,704 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:20,213 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:21,510 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:22,752 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:24,023 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:25,287 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:26,568 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:27,845 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:29,148 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:17:30,421 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 00:18:52,833 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:20:54,940 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:22:29,329 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: 'ImplementationClass' object has no attribute 'browser'
2023-12-03 00:22:29,331 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:23:30,635 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: 'ImplementationClass' object has no attribute 'browser'
2023-12-03 00:23:30,637 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:23:53,710 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: 'ImplementationClass' object has no attribute 'browser'
2023-12-03 00:23:53,711 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:24:50,803 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 00:24:50,805 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:25:23,890 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:25:23,892 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:25:50,479 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:25:50,481 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:27:02,556 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: 'PlaywrightContextManager' object has no attribute 'chromium'
2023-12-03 00:27:02,559 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:30:59,084 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 00:30:59,086 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:31:39,725 - implementation_class.py -> main_requests - line:230 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:32:25,764 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:32:52,384 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:33:24,746 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:33:24,748 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:33:55,840 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:33:57,290 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:34:30,816 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:34:30,818 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:38:18,275 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:38:18,277 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:43:45,328 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 00:44:36,144 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:45:18,229 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:57:17,484 - implementation_class.py -> main_requests - line:236 - ERROR: 请求出现错误,出现的错误是: object PlaywrightContextManager can't be used in 'await' expression
2023-12-03 00:57:17,486 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:57:31,818 - implementation_class.py -> main_requests - line:236 - ERROR: 请求出现错误,出现的错误是: object PlaywrightContextManager can't be used in 'await' expression
2023-12-03 00:57:31,819 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:59:32,387 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 00:59:58,678 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:03:02,039 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:04:04,897 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:06:00,815 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:06:23,468 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:06:29,086 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 01:06:36,989 - implementation_class.py -> main_requests - line:232 - ERROR: 请求出现错误,出现的错误是: net::ERR_NAME_NOT_RESOLVED at https://www.baidu1234.com/
2023-12-03 01:06:37,196 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu1234.com
2023-12-03 01:06:42,196 - implementation_class.py -> main_requests - line:232 - ERROR: 请求出现错误,出现的错误是: net::ERR_NAME_NOT_RESOLVED at https://www.baidu1234.com/
2023-12-03 01:06:42,406 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu1234.com
2023-12-03 01:06:50,806 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:07:58,181 - implementation_class.py -> main_requests - line:232 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:07:58,362 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:10:02,361 - implementation_class.py -> main_requests - line:232 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:10:02,547 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:16:50,126 - implementation_class.py -> main_requests - line:231 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:16:50,290 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:17:18,991 - implementation_class.py -> main_requests - line:225 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:17:19,172 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:17:44,209 - implementation_class.py -> main_requests - line:225 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:17:44,384 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:19:25,235 - implementation_class.py -> main_requests - line:225 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:19:25,398 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:20:31,721 - implementation_class.py -> main_requests - line:225 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:20:31,901 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:20:42,593 - implementation_class.py -> main_requests - line:225 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:20:42,760 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:25:13,378 - implementation_class.py -> main_requests - line:247 - ERROR: 请求出现错误,出现的错误是: ImplementationClass.add_anti_detection_js_to_context() takes 1 positional argument but 2 were given
2023-12-03 01:25:13,525 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:25:45,283 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:27:41,608 - implementation_class.py -> main_requests - line:249 - ERROR: 请求出现错误,出现的错误是: object NoneType can't be used in 'await' expression
2023-12-03 01:27:41,803 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:28:05,215 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:28:26,003 - implementation_class.py -> main_requests - line:249 - ERROR: 请求出现错误,出现的错误是: Timeout 20ms exceeded.
2023-12-03 01:28:26,250 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:29:11,541 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:29:22,908 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.baidu.com
2023-12-03 01:31:17,681 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.taobao.com
2023-12-03 01:31:52,610 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://www.taobao.com
2023-12-03 01:34:07,685 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:34:22,886 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:37:17,240 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:41:11,576 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:41:33,397 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:42:23,410 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:42:39,292 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:43:16,265 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:45:24,379 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 01:46:02,142 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:22:44,766 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:23:21,665 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:23:46,917 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:24:55,088 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:26:23,172 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:26:23,174 - rendered_api.py -> requests - line:79 - ERROR: 动态渲染请求出现错误, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:27:48,079 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:28:37,623 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?id=749746790470
2023-12-03 10:31:32,199 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:31:32,201 - rendered_api.py -> requests - line:79 - ERROR: 动态渲染请求出现错误, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:32:24,378 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:32:24,393 - rendered_api.py -> requests - line:80 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:33:17,459 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:33:17,462 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:33:50,152 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:33:50,154 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是a bytes-like object is required, not 'str'
2023-12-03 10:36:23,774 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:36:23,777 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:52:43,880 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:52:43,885 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:56:29,972 - implementation_class.py -> main_requests - line:250 - ERROR: 请求出现错误,出现的错误是: BrowserType.launch() got an unexpected keyword argument 'viewport'
2023-12-03 10:56:29,974 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:56:29,976 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 10:56:57,709 - implementation_class.py -> main_requests - line:250 - ERROR: 请求出现错误,出现的错误是: BrowserType.launch() got an unexpected keyword argument 'device'
2023-12-03 10:56:57,711 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:56:57,712 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 10:57:35,067 - implementation_class.py -> main_requests - line:250 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
Browser logs:

<launching> C:\Users\15256\AppData\Local\ms-playwright\chromium-1091\chrome-win\chrome.exe --no-sandbox --disable-gpu
<launched> pid=19868
[pid=19868][err] [1203/105714.070:ERROR:registration_protocol_win.cc(135)] TransactNamedPipe: �ܵ��ѽ����� (0x6D)
[pid=19868][err] [19868:16996:1203/105714.210:ERROR:policy_logger.cc(156)] :components\enterprise\browser\controller\chrome_browser_cloud_management_controller.cc(161) Cloud management controller initialization aborted as CBCM is not enabled. Please use the `--enable-chrome-browser-cloud-management` command line flag to enable it if you are not using the official Google Chrome build.
2023-12-03 10:57:35,070 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:57:35,072 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 10:58:13,137 - implementation_class.py -> main_requests - line:250 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
Browser logs:

<launching> C:\Users\15256\AppData\Local\ms-playwright\chromium-1091\chrome-win\chrome.exe --disable-gpu
<launched> pid=16512
[pid=16512][err] [16512:19308:1203/105802.963:ERROR:policy_logger.cc(156)] :components\enterprise\browser\controller\chrome_browser_cloud_management_controller.cc(161) Cloud management controller initialization aborted as CBCM is not enabled. Please use the `--enable-chrome-browser-cloud-management` command line flag to enable it if you are not using the official Google Chrome build.
2023-12-03 10:58:13,140 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:58:13,141 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 10:58:37,484 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:58:37,487 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:58:58,229 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:58:58,233 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 10:59:53,302 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png
2023-12-03 10:59:53,309 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: https://ucc.alicdn.com/images/user-upload-01/0360a32d973d4144841f93acd679a449.png, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 11:00:07,801 - implementation_class.py -> main_requests - line:250 - ERROR: 请求出现错误,出现的错误是: Protocol error (Page.navigate): Cannot navigate to invalid URL
2023-12-03 11:00:07,802 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: www.google.com
2023-12-03 11:00:07,803 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: www.google.com, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 11:00:31,794 - rendered_api.py -> requests - line:71 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:00:31,796 - rendered_api.py -> requests - line:81 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 11:05:15,855 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:05:15,859 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 11:05:55,424 - implementation_class.py -> main_requests - line:253 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:05:55,430 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:05:55,432 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 11:06:13,634 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:06:13,637 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 11:06:28,882 - implementation_class.py -> main_requests - line:253 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:06:28,886 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:06:28,888 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 11:08:14,064 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:08:14,067 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:08:14,068 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 11:08:38,290 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:08:38,292 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是Object of type bytes is not JSON serializable
2023-12-03 11:08:51,653 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:08:51,655 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:08:51,657 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'NoneType'
2023-12-03 11:09:14,144 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:09:14,146 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'str'
2023-12-03 11:09:38,226 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:09:38,227 - rendered_api.py -> requests - line:85 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.google.com, 错误内容是a bytes-like object is required, not 'str'
2023-12-03 11:10:21,642 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:11:58,169 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:11:58,171 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:12:10,606 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:12:10,607 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:12:15,074 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:12:15,075 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:12:36,678 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:12:36,685 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:13:05,672 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:13:31,914 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:13:31,915 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:13:54,000 - implementation_class.py -> main_requests - line:255 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 11:13:54,001 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:14:13,065 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:14:43,679 - implementation_class.py -> main_requests - line:253 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:14:43,682 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:15:06,837 - implementation_class.py -> main_requests - line:253 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:15:06,839 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:16:15,521 - implementation_class.py -> main_requests - line:253 - ERROR: 请求出现错误,出现的错误是: Timeout 1000ms exceeded.
2023-12-03 11:16:15,522 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:16:48,777 - rendered_api.py -> requests - line:75 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:29:43,652 - implementation_class.py -> main_requests - line:265 - ERROR: 请求出现错误,出现的错误是: object NoneType can't be used in 'await' expression
2023-12-03 11:29:43,654 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:30:23,750 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.google.com
2023-12-03 11:31:00,016 - implementation_class.py -> main_requests - line:265 - ERROR: 请求出现错误,出现的错误是: net::ERR_HTTP_RESPONSE_CODE_FAILURE at http://www.taobao.com.com/
2023-12-03 11:31:00,038 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com.com
2023-12-03 11:31:19,555 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:32:10,488 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:33:47,970 - implementation_class.py -> main_requests - line:265 - ERROR: 请求出现错误,出现的错误是: Timeout 10000ms exceeded.
2023-12-03 11:33:47,973 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:34:39,373 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:35:09,944 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:37:02,947 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:37:25,429 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:37:44,005 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:39:05,510 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:39:44,300 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:40:14,078 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:45:24,822 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:45:53,964 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:46:46,935 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:48:28,907 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:49:16,247 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:52:33,762 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:53:31,540 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:55:38,790 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 11:58:42,907 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:00:18,512 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:01:53,674 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:02:29,293 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:05:00,714 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:06:28,682 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:10:41,423 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:22:40,476 - implementation_class.py -> main_requests - line:273 - ERROR: 请求出现错误,出现的错误是: 'Page' object has no attribute 'wait_for'
2023-12-03 12:22:40,478 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:40:51,302 - implementation_class.py -> main_requests - line:273 - ERROR: 请求出现错误,出现的错误是: object NoneType can't be used in 'await' expression
2023-12-03 12:40:51,304 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:41:41,803 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:43:25,384 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:48:37,695 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:48:53,314 - implementation_class.py -> main_requests - line:273 - ERROR: 请求出现错误,出现的错误是: Timeout 10000ms exceeded.
2023-12-03 12:48:53,315 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:49:10,652 - implementation_class.py -> main_requests - line:273 - ERROR: 请求出现错误,出现的错误是: Timeout 10000ms exceeded.
2023-12-03 12:49:10,655 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:49:33,455 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:49:40,850 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:51:14,660 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:53:31,604 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:55:47,695 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:56:26,909 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:57:23,298 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 12:59:29,346 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:03:23,346 - implementation_class.py -> main_requests - line:295 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:03:23,352 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:03:46,584 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:05:17,710 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:05:17,717 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:05:26,687 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:05:50,553 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:05:50,557 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:06:38,061 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:06:56,640 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:06:56,773 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:07:05,835 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:07:14,760 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:07:33,389 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:08:52,637 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:08:52,786 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:09:11,438 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:09:11,565 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:09:32,948 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 13:09:33,078 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:10:01,493 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:10:37,291 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 20000ms exceeded.
2023-12-03 13:10:37,397 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:11:02,843 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 20000ms exceeded.
2023-12-03 13:11:02,950 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:11:27,912 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 20000ms exceeded.
2023-12-03 13:11:28,031 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:12:17,227 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 25000ms exceeded.
2023-12-03 13:12:17,355 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:13:06,230 - implementation_class.py -> main_requests - line:296 - ERROR: 请求出现错误,出现的错误是: Timeout 25000ms exceeded.
2023-12-03 13:13:06,337 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:13:31,392 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:13:40,961 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 13:21:53,591 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:23:05,056 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:23:22,718 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:25:44,192 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:25:57,353 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:27:27,244 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:27:41,607 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:28:03,001 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:28:13,296 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:28:44,842 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:28:56,249 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:29:34,814 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:29:59,333 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:56:38,921 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:56:50,347 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:57:01,347 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 13:57:14,403 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:01:51,207 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:02:21,331 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:03:43,006 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:06:41,350 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:06:41,351 - rendered_api.py -> requests - line:87 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'list' object has no attribute 'replace'
2023-12-03 14:07:03,423 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:11:25,368 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:16:41,386 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 14:18:53,433 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 14:22:14,585 - implementation_class.py -> main_requests - line:300 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 14:22:14,586 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 14:22:36,232 - implementation_class.py -> main_requests - line:300 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 14:22:36,233 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 14:23:36,856 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 14:24:22,005 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:25:01,155 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:26:25,191 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:43:07,596 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:43:23,435 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:46:16,077 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:46:51,722 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:48:21,942 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:48:32,629 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:52:39,175 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:53:34,417 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:59:05,074 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Timeout 25000ms exceeded.
2023-12-03 14:59:05,078 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 14:59:46,834 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Target page, context or browser has been closed
2023-12-03 14:59:46,841 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 15:01:07,027 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Execution context was destroyed, most likely because of a navigation.
2023-12-03 15:01:07,029 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 15:41:00,341 - rendered_api.py -> requests - line:78 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 15:51:05,796 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 18:41:58,117 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 18:43:35,630 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 18:43:53,019 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:43:53,020 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:43:54,899 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:43:54,901 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:43:56,157 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:43:56,158 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:43:57,139 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:43:57,140 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:43:59,660 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:43:59,662 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:06,562 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:06,563 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:07,718 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:07,719 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:08,579 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:08,580 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:09,319 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:09,320 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:09,732 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:09,732 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:10,275 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:10,277 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:30,986 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:30,987 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:32,293 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:32,294 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:39,748 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:39,748 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:44:40,724 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:44:40,725 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:10,403 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:10,404 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:15,533 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:15,534 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:16,428 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:16,429 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:51,071 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:51,073 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:52,275 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:52,276 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:53,392 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:53,394 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:54,268 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:54,270 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:55,118 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:55,119 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:55,979 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:55,981 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:57,144 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:57,146 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:45:58,064 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:45:58,065 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: http://www.taobao.com, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:07,267 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:07,268 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:31,181 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:31,183 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:32,057 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:32,058 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:32,863 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:32,865 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:33,427 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:33,428 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:33,831 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:33,832 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:34,282 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:34,283 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:34,732 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:34,733 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:35,158 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:35,159 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:35,990 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:35,992 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:46:36,815 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:46:36,816 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:11,921 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:11,923 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:13,116 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:13,118 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:13,947 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:13,949 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:14,864 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:14,865 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:15,741 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:15,742 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:16,548 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:16,550 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:17,293 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:17,294 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:25,741 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:25,743 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:26,734 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:26,735 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:36,435 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:36,437 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:47:37,307 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 18:47:37,308 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 18:48:00,003 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 18:48:07,168 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 18:48:18,005 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 18:48:24,517 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 18:54:52,224 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Timeout 15000ms exceeded.
2023-12-03 18:54:52,323 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 18:55:03,292 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 18:55:27,735 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 18:59:55,163 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: http://www.taobao.com
2023-12-03 19:03:33,604 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 19:03:57,178 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 19:04:08,551 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc
2023-12-03 19:05:16,975 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: 
2023-12-03 19:05:16,977 - rendered_api.py -> requests - line:90 - ERROR: 动态渲染请求出现错误, 请求的url是: https://www.xiaohongshu.com/explore/654ed925000000001b00c2bc, 错误内容是'NoneType' object has no attribute 'close'
2023-12-03 20:07:42,800 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:08:04,811 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:15:01,756 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Object of type CookiesItem is not JSON serializable
2023-12-03 20:15:01,758 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:16:14,155 - implementation_class.py -> main_requests - line:304 - ERROR: 请求出现错误,出现的错误是: Object of type CookiesItem is not JSON serializable
2023-12-03 20:28:37,631 - implementation_class.py -> main_requests - line:316 - ERROR: 请求出现错误,出现的错误是: Object of type CookiesItem is not JSON serializable
2023-12-03 20:28:37,633 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:28:55,761 - implementation_class.py -> main_requests - line:316 - ERROR: 请求出现错误,出现的错误是: Object of type CookiesItem is not JSON serializable
2023-12-03 20:28:55,762 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:30:16,047 - implementation_class.py -> main_requests - line:316 - ERROR: 请求出现错误,出现的错误是: Object of type CookiesItem is not JSON serializable
2023-12-03 20:30:16,048 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:32:53,304 - implementation_class.py -> main_requests - line:306 - ERROR: 请求出现错误,出现的错误是: 'CookiesItem' object has no attribute 'to_dict'
2023-12-03 20:32:53,306 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:33:41,466 - implementation_class.py -> main_requests - line:306 - ERROR: 请求出现错误,出现的错误是: Cookie should have a url or a domain/path pair
2023-12-03 20:33:41,468 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:35:07,262 - implementation_class.py -> main_requests - line:306 - ERROR: 请求出现错误,出现的错误是: Cookie should have a url or a domain/path pair
2023-12-03 20:35:07,263 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:41:42,943 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:42:07,221 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:43:32,053 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:46:17,497 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
2023-12-03 20:52:29,629 - implementation_class.py -> main_requests - line:308 - ERROR: 请求出现错误,出现的错误是: Cookie should have a url or a domain/path pair
2023-12-03 20:52:29,631 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 20:54:34,013 - implementation_class.py -> main_requests - line:308 - ERROR: 请求出现错误,出现的错误是: Cookie should have a url or a domain/path pair
2023-12-03 20:54:34,014 - rendered_api.py -> requests - line:79 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 21:33:42,741 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:04:36,820 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:05:02,019 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:06:04,551 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:06:28,317 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:06:39,499 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:07:54,069 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://item.taobao.com/item.htm?spm=a21bo.jianhua.201876.1.5af92a89EP5tnC&id=36903311829&scm=1007.40986.276750.0&pvid=24189f20-0708-421d-980f-730cd37cfc05
2023-12-03 22:08:38,691 - rendered_api.py -> requests - line:80 - INFO: 动态渲染请求成功,请求的url是: https://www.baidu.com
